utils/url.test.tsx

```diff
@@ -42,7 +42,7 @@ describe('Utils.URL', () => {
             },
         ];
 
-        testCases.forEach((testCase) => it(testCase.description, () => {
+        testCases.forEach((testCase) => it(testCase.description, () => { // eslint-disable-line max-nested-callbacks
```

Comment:
This can be removed with the latest master branch, the `eslintrc.json` has been updated

Reply:
Excellent, that's very helpful - thanks for pointing out. I updated the PR.